### ISSUE TITLE
Error running plugin google ReferencError: search is not defined.

### ISSUE THREAD

**Issue by mijoharas** [KELiON/cerebro#167]
I seem to be having problems with a lot of the plugins not working.

```
Error running plugin google ReferenceError: search is not defined
    at Object.googlePlugin [as fn] (file:///usr/lib/cerebro/resources/app.asar/dist/main.bundle.js:29247:14)
    at Object.keys.forEach.error (file:///usr/lib/cerebro/resources/app.asar/dist/main.bundle.js:114849:32)
    at Array.forEach (native)
    at eachPlugin (file:///usr/lib/cerebro/resources/app.asar/dist/main.bundle.js:114847:35)
    at dispatch (file:///usr/lib/cerebro/resources/app.asar/dist/main.bundle.js:114905:6)
    at file:///usr/lib/cerebro/resources/app.asar/dist/main.bundle.js:123521:19
    at Object.onChange (file:///usr/lib/cerebro/resources/app.asar/dist/main.bundle.js:123608:13)
    at onChange (file:///usr/lib/cerebro/resources/app.asar/dist/main.bundle.js:115880:34)
    at Object.executeOnChange (file:///usr/lib/cerebro/resources/app.asar/dist/main.bundle.js:11257:35)
    at ReactDOMComponent._handleChange (file:///usr/lib/cerebro/resources/app.asar/dist/main.bundle.js:101930:39)
```

In addition, I have tried installing some plugins but they do not seem to be registered.

**Comment by KELiON** [comment]
@mijoharas what version do you use?

**Comment by mijoharas** [comment]
Your are using latest version of Cerebro (0.2.6).

(installed from arch `cerebro-git` aur package).

**Comment by maximbaz** [comment]
I didn't notice it earlier, but I reproduce this as well. I think it was caused by https://github.com/KELiON/cerebro/pull/157

Can't start apps anymore, and google plugin is not there.

**Comment by KELiON** [comment]
@maximbaz looks like it is something related to babel or other building stuff because I can reproduce it only in production build. In dev. environment everything works well. So, I'm investigating what is the reason

**Comment by maximbaz** [comment]
Yes, I just reached the same conclusion, it is not reproducing with `yarn run dev`. Interesting...

**Comment by KELiON** [comment]
@maximbaz hm, ok. If I change `const` [here](https://github.com/KELiON/cerebro/blob/master/app/main/plugins/core/google/index.js#L21) to `var` it works in production build too. Current status:

![image](https://cloud.githubusercontent.com/assets/594298/23342944/8ce95b68-fc63-11e6-8a81-992b11628ae4.png)

Going to use it as a hotfix now. I have no idea what causes this issue, but I'm almost sure that the reason in #148, because I've changed a bit building process there.

**Comment by KELiON** [comment]
@maximbaz I just pushed the fix, but should it be updated somehow for `cerebro-git`, or it just uses  latest version from master branch?

@mijoharas I think you can try to reinstall it

**Comment by maximbaz** [comment]
It just uses the latest code from `master` branch, no extra change required. Just tested, the search is working again. What a weird issue!

**Comment by mijoharas** [comment]
Hi, reinstalling fixes the search issue.

I am still having troubles with the plugins though. The only other error message I can find is:

```
main.bundle.js:95675 The provided value 'moz-chunked-arraybuffer' is not a valid enum value of type XMLHttpRequestResponseType.
```

**Comment by maximbaz** [comment]
what kind of troubles do you have?

**Comment by mijoharas** [comment]
Hi, @maximbaz all of the plugins I have tried installing do not work. `devdocs` `lipsum` and `linux` so far are ones I've tested. (tried typing in `devdocs`, `lipsum` and `shutdown` to test the respective plugins, but nothing but google search and translate seems to come up).

The plugins appear to have installed themselves, and when I install a new plugin nothing seems untoward in either of the consoles.
